packages/nimble-components/src/wafer-map/specs/features/grid-dimension.md

```diff
@@ -0,0 +1,28 @@
+# Nimble Wafer Map Grid Dimensions
+
+## Problem Statement
+
+The Wafer Map displays dies on a canvas inside a circular border based on an internal grid which is calculated based on the data input.
+This approach is limiting the performance of the wafer and the grid dimensions can be erroneous if the displayed data is just a subset of a larger data set.
+To mitigate these problems, we will reveal three new API inputs which will allow the users to set the grid dimensions themselves, thus saving processing time and correcting inconsistencies generated by the lack of data context.
+
```

Comment:
Can you give a more concrete / very specific examples of  how the user of the wafer map would leverage these apis?

Like snippets of API usage and what it accomplishes.

In particular I'm trying to understand the component workflow and how the API is expected to be used by applications.

Reply:
I provided a visual representation of what is expected. if there should be more examples please let me know

Reply:
I appreciate you including the screenshot but I think I'm even more confused about what this feature is trying to accomplish. 

Like I don't know what I'm supposed to takeaway from this screenshot:
![image](https://github.com/ni/nimble/assets/1588923/cf5c0115-8add-40b7-8cbf-6b404d30798b)

Is that a highlighted row? where is the actual wafer data? should the outer part of the wafer map still be shown?

You may need to reach out to help us understand what's going on clearer. If you can catch @jattasNI or I to help understand the goals and expected visual / user interaction impact of the changes that would help to make sure the details are captured in the spec.

Reply:
@munteannatan I would encourage you to try and reach out and resolve these questions sooner than later depending on your timelines.

As an FYI Jesse will be OOO for the next two weeks and we both have NI-Tech commitments. So be aware that response times will be longer as we approach NI-Tech and bandwidth is reduced.